README.md

```diff
@@ -58,7 +58,8 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-#### <a href="Coming Soon">Coming Soon</a>
+math-magicians-quotes
+<a href="#">Coming soon</a>
 
```

Comment:
- [OPTIONAL]  Your project topic `Math-magicians`  is duplicated at the bottom of your readme for, after the license section,   it will be nice to delete as this will make your readme file clean and professional, it will show you have attention to details.